.github/workflows/ci.test.yml

```
@@ -1,24 +1,24 @@
 name: Test
 
-on: 
-  pull_request:
-  push:
-    branches:
-      - 'develop'
+on:
+    pull_request:
+    push:
+        branches:
+            - 'develop'
 
 jobs:
-  shared:
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v2
+    shared:
+        runs-on: ubuntu-18.04
```

I think this has been forgotten :relaxed: 

```suggestion
        runs-on: ubuntu-20.04
```